CHANGELOG.md

```diff
@@ -6,22 +6,29 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Added
```

Comment:
Something wrong with the changelog?

Reply:
ah no, entries have been moved.

Reply:
Yes, it's good to have `Added` at the top.